### ISSUE TITLE
[SF][Table][A11y]: accessible name of button is missing when cell focused

### ISSUE THREAD

**Issue by tandan0813** [UI5/webcomponents#13216]
### Bug Description

When a table cell contains an icon button, and when focus is placed on that cell, the button's accessible name is not announced by the screen reader — only 'button' is read out."

<img width="2339" height="858" alt="Image" src="https://github.com/user-attachments/assets/2021bae2-cee0-4708-97f3-545ecdf37208" />

### Expected Behaviour

Info icon button needs to have an accessible name when the cell is focussed

### Isolated Example

https://stackblitz.com/edit/github-h8uvdu2w?file=src%2FApp.tsx

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

2.18.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by aborjinik** [comment]
hello @UI5/ui5-team-ctr-bal team, it seems like `accessibilityInfo` of the `Button` does not include the `accessibleName` could you please take a look?
